Listing '.'...
Listing './bin'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/marcio0_django_piston-0.2.5.1.egg-info'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/piston'...
Compiling './lib/python3.5/site-packages/piston/authentication.py'...
***   File "./lib/python3.5/site-packages/piston/authentication.py", line 97
    except ImportError, e:
                      ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/piston/emitters.py'...
***   File "./lib/python3.5/site-packages/piston/emitters.py", line 264
    except NoReverseMatch, e:
                         ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/piston/fixtures'...
Compiling './lib/python3.5/site-packages/piston/resource.py'...
***   File "./lib/python3.5/site-packages/piston/resource.py", line 34
    raise AttributeError, "Handler not callable."
                        ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/piston/templates'...
Listing './lib/python3.5/site-packages/piston/templates/piston'...
Compiling './lib/python3.5/site-packages/piston/utils.py'...
***   File "./lib/python3.5/site-packages/piston/utils.py", line 339
    print "Mail being sent, to=%s" % consumer.user.email
                                 ^
SyntaxError: Missing parentheses in call to 'print'

Compiling './lib/python3.5/site-packages/piston/validate_jsonp.py'...
***   File "./lib/python3.5/site-packages/piston/validate_jsonp.py", line 206
    """
     ^
SyntaxError: (unicode error) 'unicodeescape' codec can't decode bytes in position 709-712: truncated \uXXXX escape

Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './share'...
Listing './share/python-wheels'...
